coldfront/core/project/views.py

```diff
@@ -604,6 +612,24 @@ def post(self, request, *args, **kwargs):
                     users_already_in_project.append(ele)
             context['users_already_in_project'] = users_already_in_project
 
+        allocations_with_eula = {}
```

Comment:
Should we guard here with `if ALLOCATION_EULA_ENABLE:`? it's likely not needed if this is not enabled.

Reply:
Added guard and removed code that did not appear to be used anywhere